hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieCDCLogger.java

```diff
@@ -140,7 +140,8 @@ public HoodieCDCLogger(
           new DefaultSizeEstimator<>(),
           config.getCommonConfig().getSpillableDiskMapType(),
           new DefaultSerializer<>(),
-          config.getCommonConfig().isBitCaskDiskMapCompressionEnabled());
+          config.getCommonConfig().isBitCaskDiskMapCompressionEnabled(),
+          getClass().getSimpleName());
```

Comment:
The error stack trace should already include the caller class name, I don't think we need this and people don't write debug code by modifying the top-API.

Reply:
Infact, I do differentiate for certain callers 
```
getClass().getSimpleName() + "_fileIdToPendingClusteringMap"
```

So, if class name alone is sufficient, I use that as is. If not, if same class has multiple usages, I suffix for each usage.

Reply:
bad code cinvention I would say.